playground/resolve/index.html

```diff
@@ -378,6 +381,8 @@ <h2>resolve package that contains # in path</h2>
     '.path-contains-sharp-symbol',
     `[success] ${contains.call('#', '#')} ${last.call('#')}`,
   )
+
+  text('.mjs-extension-with-query', tsMjsExtensionWithQueryMsg)
```

Comment:
I guess the diff in this file is not needed. (or we can fetch the content here and check that in the test)